aidants_connect_web/templates/public_website/habilitation.html

```diff
@@ -75,8 +72,43 @@ <h2>Liste des structures éligibles à l’utilisation d’Aidants Connect</h2>
             Exemples : les travailleurs sociaux.
           </li>
         </ul>
-        <p><strong>Les aidants familiaux, les bénévoles et services civiques ne sont donc pas éligibles.</strong></p>
+        <p><strong>Les aidants familiaux, les bénévoles et services civiques ne sont donc pas éligibles. Pour plus
+          d'informations,
+          nous vous invitons à consulter <a href="https://aidantsconnect.beta.gouv.fr/faq/mandat/">notre
```

Comment:
Si on merge la PR #330 avant celle-ci, on pourra mettre ceci à la place : 
```suggestion
          nous vous invitons à consulter <a href="{% faq_habilitation %}">notre
```